plugin_III/game_III/CPlayerInfo.h

```diff
@@ -60,8 +60,8 @@ class CPlayerInfo {
     int field_110;
     bool m_bInfiniteSprint;
     bool m_bFastReload;
-    bool m_bGetOutOfJailFree;
-    bool m_bGetOutOfHospitalFree;
+    bool m_bGetOutOfJailFree;                   // Player doesn't lose money/weapons next time arrested
+    bool m_bFreeHealthCare;                     // Player doesn't lose money nexed time patched up at hospital
```

Comment:
I renamed III since MTA has a similar name [bFreeHealthCare](https://github.com/multitheftauto/mtasa-blue/blob/51acb72cc7002d8c9da13abf4262ad96378f3eff/Client/game_sa/CPlayerInfoSA.h#L235). But I must agree `m_bGetOutOfHospitalFree` provides clearer context.